src/params/tech_params.yaml

```diff
@@ -366,28 +354,7 @@ latency: # ns
     UAdd: 0.47
     USub: 0.47
     OffChipIO: 0.0
-latency_scale:
-  512: 1
-  1024: 2
-  2048: 3
-  4096: 4
-  8192: 5
-  16384: 6
-  32768: 7
-  65536: 8
-  131072: 9
-  262144: 10
-  524288: 11
-  1048576: 12
-  2097152: 13
-  4194304: 14
-  8388608: 15
-  16777216: 16
-  33554432: 17
-  67108864: 18
-  134217728: 19
-  268435456: 20
-  536870912: 21
+
```

Comment:
are we replacing latency/power scaling with something to account for different size-dependent costs?

Reply:
Cacti Integration handles this.